tokenizers/src/pre_tokenizers/byte_level.rs

```diff
@@ -12,7 +12,7 @@ use crate::utils::macro_rules_attribute;
 
 /// Converts bytes to unicode characters.
 /// See https://github.com/openai/gpt-2/blob/master/src/encoder.py#L9
-pub(crate) fn bytes_char() -> AHashMap<u8, char> {
+pub fn bytes_char() -> AHashMap<u8, char> {
```

Comment:
Why is this made `pub`, this seems like a mistake. In fact, it seems even the previous `pub(crate)` is misplaced. Can you make this private and reuse the `CHAR_BYTES` static in this module by making it `pub(crate)`?

Reply:
Yes, but `normalizers/byte_level.rs` only needs the function to build an exact copy of the static `BYTE_TO_CHAR`. So as a drive-by cleanup, we could make just the statics `pub(crate)` and reuse them across both `pre_tokenizers/byte_level.rs` and `normalizers/byte_level.rs`. We can then make this function private.